drivers/windows/file_access_windows.cpp

```diff
@@ -61,6 +61,11 @@ void FileAccessWindows::check_errors() const {
 bool FileAccessWindows::is_path_invalid(const String &p_path) {
 	// Check for invalid operating system file.
 	String fname = p_path.get_file().get_basename().to_lower();
+
+	int dot = fname.find(".");
+	if (dot != -1) {
+		fname = fname.substr(0, dot);
+	}
```

Comment:
```suggestion
	String fname = p_path.get_file().to_lower();

	int dot = fname.find(".");
	if (dot != -1) {
		fname = fname.substr(0, dot);
	}
```

So I guess something like this might be more correct, and avoid redundant search.